tests/test_server_config.py

```diff
@@ -972,11 +972,15 @@ async def test_server_proto_configure_describe_system_config(self):
             conf4 = "CONFIGURE INSTANCE SET memprop := <cfg::memory>'100MiB';"
             await self.con.execute(conf4)
 
+            conf5 = "CONFIGURE INSTANCE SET enumprop := <cfg::TestEnum>'Two';"
```

Comment:
Can you also add a test that `cfg::TestEnum.Two` works?

Reply:
Unresolving this thread -- can you add this test in this commit?

Reply:
No, because we cannot make `DESCRIBE INSTANCE CONFIG` return a non-casting version.  There is a dot-notation test elsewhere.